### ISSUE TITLE
Tooltip in running text is not accessible for keyboard users

### ISSUE THREAD

**Issue by michael-arvanitidis** [digdir/designsystemet#1871]
### Description of the bug

The tooltip in this example https://storybook.designsystemet.no/iframe.html?args=&id=komponenter-tooltip--complex&viewMode=story is only shown on hover on the `<abbr>`-tag. Keyboard users will not be able to trigger the popover. 

### Steps To Reproduce

1. Go to https://storybook.designsystemet.no/iframe.html?args=&id=komponenter-tooltip--complex&viewMode=story
2. Try to trigger the popover using the keyboard

### Additional Information

Setting `tabindex="0"` on the `<abbr>` will make this focusable. However the usage of the `<abbr>`-tag might not be the best element to use for tooltips. For more info see this page https://developer.mozilla.org/en-US/docs/Web/HTML/Element/abbr

**Comment by Barsnes** [comment]
We have for now decided to remove this story in #2130.
`tabindex` is meant for interactive elements, and there are more issues with tooltips in text, that is not solved by changing the semantic element, or any attributes on it.
